Tests.Integration.Presentation.Web/Organizations/V2/OrganizationGridConfigurationAPIV2Test.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.API.V1;
+using Presentation.Web.Models.API.V2.Internal.Response.Organizations;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.Organizations.V2
+{
+    public class OrganizationGridConfigurationApiV2Test : WithAutoFixture
+    {
+
+        [Fact]
+        public async Task LocalAdminCanSaveGridConfig()
+        {
+            var (org, cookie) = await CreatePrerequisites();
+
+            var columns = CreateTestColumns().ToList();
+            var saveResponse = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, cookie);
+            Assert.Equal(HttpStatusCode.OK, saveResponse.StatusCode);
+        }
+
+        [Fact]
+        public async Task RegularUserCanGetGridConfiguration()
+        {
+            //Admins saves a column configuration
+            var columns = CreateTestColumns().ToArray();
+            var (org, localAdminCookie) = await CreatePrerequisites();
+            _ = await OrganizationGridConfigurationTestHelper.SendSaveConfigurationRequestAsync(org.Uuid, columns, localAdminCookie);
```

Comment:
use the newly created SaveConfigurationRequestAsync here (you'll get the OK check for free)

Reply:
also no need to use the discard token, just don't assign the method to anything